modules/webapp/webapp.bicep

```diff
@@ -126,6 +134,20 @@ resource webApp 'Microsoft.Web/sites@2022-03-01' = {
 }
 
 
+resource webSiteName_web 'Microsoft.Web/sites/sourcecontrols@2015-08-01' = {
```

Comment:
Is `2015-...` the latest available API version?

Reply:
updated